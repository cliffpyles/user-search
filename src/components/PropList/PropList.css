.PropList {
  display: flex;
  flex-flow: row wrap;
}

.PropList li {
  display: flex;
  list-style-type: none;
  flex: 1 0 50%;
}

.PropList strong {
  display: block;
  padding: 0 0.25em;
}

.PropList span {
  display: block;
  padding: 0 0.25em;
}
