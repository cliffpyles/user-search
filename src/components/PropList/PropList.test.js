import React from 'react'
import PropList from './'

test('can render a PropList', () => {
  const data = {
    example1: 'value',
    example2: 'value',
    example3: 'value'
  }
  const component = render(<PropList data={data} allowedProps={['example1']} />)

  expect(component.innerHTML).toMatchSnapshot()
})
