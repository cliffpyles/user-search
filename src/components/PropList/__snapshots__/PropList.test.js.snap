// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a PropList 1`] = `"<ul class=\\"PropList\\"><li><strong>example1</strong><span>value</span></li></ul>"`;
